From 42ee55a273331dc1edcfcd8baf415f694e1d254d Mon Sep 17 00:00:00 2001
From: Daron Chabot <chabot@frib.msu.edu>
Date: Thu, 18 May 2017 10:38:31 -0400
Subject: [PATCH] MNT: hardcode version

---
 epics/__init__.py | 4 +---
 setup.py          | 5 ++---
 2 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/epics/__init__.py b/epics/__init__.py
index c6fc313..a2b65d9 100644
--- a/epics/__init__.py
+++ b/epics/__init__.py
@@ -1,6 +1,4 @@
-from ._version import get_versions
-__version__ = get_versions()['version']
-del get_versions
+__version__ = '3.3.0'
 
 __doc__ = """
    epics channel access python module
diff --git a/setup.py b/setup.py
index cbd4ae0..00adb3e 100644
--- a/setup.py
+++ b/setup.py
@@ -7,7 +7,7 @@ import sys
 import epics
 import shutil
 
-import versioneer
+import epics
 
 long_desc = '''Python Interface to the Epics Channel Access protocol
 of the Epics control system.   PyEpics provides 3 layers of access to
@@ -57,8 +57,7 @@ if PY_MAJOR == 2 and PY_MINOR < 6:
                 pjoin('epics', 'utils3.py'))
 
 setup(name = 'pyepics',
-      version = versioneer.get_version(),
-      cmdclass = versioneer.get_cmdclass(),
+      version = epics.__version__,
       author       = 'Matthew Newville',
       author_email = 'newville@cars.uchicago.edu',
       url          = 'http://pyepics.github.io/pyepics/',
-- 
2.1.4

